app/code/Magento/Integration/Model/CustomerTokenService.php

```diff
@@ -84,12 +84,12 @@ public function __construct(
     /**
      * @inheritdoc
      */
-    public function createCustomerAccessToken($username, $password)
+    public function createCustomerAccessToken($username, $password, $websiteId)
```

Comment:
The `$websiteId` parameter must be optional for backward compatibility.

Reply:
We are not able to change public methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
@dmytro-ch thanks for explaination i will update it according to Backward Compatible Development Guide.

Reply:
@dmytro-ch  & @sidolov could you help me to fix Semantic Version Checker build regarding issue. Where will i have to update for fixing this.

Reply:
Hi @Shubham-Webkul , Semantic Version Checker indicates the problem with the provided solution. Please, check my previous comment

Reply:
@sidolov I have made parameter optional still got an issue. Will i have to create another method. Can be update public method parameters?